src/changelog/unreleased.md

```diff
@@ -40,3 +40,4 @@
 - **Breaking:** Removed `EventLoopBuilder::with_user_event`, the functionality is now available in `EventLoop::with_user_event`.
 - Add `Window::default_attributes` to get default `WindowAttributes`.
 - `log` has been replaced with `tracing`. The old behavior can be emulated by setting the `log` feature on the `tracing` crate.
+- On Windows, confine cursor to center of window when grabbed and hidden.
```

Comment:
Same here. Configure your editor.

Reply:
Thanks, done in 2f313c1ec433390e3f97868c149b882744e0d2cd.

I'm not usually on Windows so haven't setup my development environment there, will make sure to do so when/if I do anything more Windows specific.